dubbot.md

```diff
@@ -0,0 +1,22 @@
+# Dubbot
+
+## Accessing dubbot
+
+1. Go to https://princeton.dubbot.com
+1. Select the group "Discovery & Access Services"
+
+
+## Crawling
+
+By default, dubbot crawls the catalog every 90 days.  However, we can reach out to John Jameson or Rachel Busnardo to crawl more frequently,
+or to do an ad-hoc crawl.
+
+If there is one specific page, we'd like to check again, we can open it in dubbot and press the "Crawl" button.  It may spin forever, but
+if you reload the page, it should show you the new results.
```

Comment:
Another hard return

```suggestion
If there is one specific page, we'd like to check again, we can open it in dubbot and press the "Crawl" button.  It may spin forever, but if you reload the page, it should show you the new results.
```